package.json

```diff
@@ -8,6 +8,11 @@
     "engines": {
         "vscode": "^1.5.0"
     },
+    "extensionKind": [
+        "ui",
+        "workspace",
```

Comment:
I don't think we should have `"workspace"` listed. From the docs (emphasis mine):

> - UI Extensions: These extensions contribute to the VS Code user interface and are always run on the user's local machine. UI Extensions cannot directly access files in the remote workspace, or run scripts/tools installed in that workspace or on the machine. Example UI Extensions include: themes, snippets, **language grammars**, and keymaps.
> - Workspace Extensions: These extensions are run on the same machine as where the workspace is located. When in a local workspace, Workspace Extensions run on the local machine. When in a remote workspace or when using Codespaces, Workspace Extensions run on the remote machine / environment. Workspace Extensions can access files in the workspace to provide rich, multi-file language services, debugger support, or perform complex operations on multiple files in the workspace (either directly or by invoking scripts/tools). While Workspace Extensions do not focus on modifying the UI, they can contribute explorers, views, and other UI elements as well.

I don't think we should be running the highlighter on a remote server, for instance. It should only run on the local machine.

Reply:
Ok